string name
server_slam/Pose pose
---
